onadata/libs/filters.py

```diff
@@ -28,13 +28,13 @@
 User = get_user_model()
 
 
-def _is_public_xform(export_id: int):
+def _public_export_id_or_none(export_id: int):
```

Comment:
Should this be `_public_xform_id_or_none(export_id: int)`? The function would return the `xform_id` for the `XForm` in the given `Export` if it is a public `XForm`.